Modules/xSharePoint/DSCResources/MSFT_xSPFarmAdministrators/MSFT_xSPFarmAdministrators.psm1

```diff
@@ -0,0 +1,224 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)] [System.String] $Name,
+        [parameter(Mandatory = $false)] [System.String[]] $Members,
+        [parameter(Mandatory = $false)] [System.String[]] $MembersToInclude,
```

Comment:
Also its worth noting that this approach lines up with the [out of the box Group resource](https://msdn.microsoft.com/en-us/PowerShell/DSC/groupResource) so I'm happy to see us use the same approach